content/docs/user-guide/build/distributable-engine.md

```diff
@@ -0,0 +1,226 @@
+                    "LoadLevel": "MyLevel"
+                }
+            }
+        }
+    }
+    ```
+
+To verify that the level loads when the game is launched, run the project's game launcher:
+
+```cmd
+W:\MyProject\build\windows_vs2019\bin\profile\MyProject.GameLauncher.exe
+```
+
+## Create a distributable build
+
+The final step to create the binaries for distribution to project developers is to create the set of install binaries that they can use without building the engine from source. Like you did to create the local binaries for the engine, you'll now create binaries for the project.
+
+From the project directory, run the following steps:
+
+1. Create an install build of the project.
+    
+    ```cmd
+    cmake --build build\windows_vs2019 --config profile --target INSTALL
+    ```
+
+    This creates the binaries in an `install` subdirectory of your project. For our example it would be `W:\MyProject\install`.
+
+## Distribute your build
+
+At this point, the created `install` directory is your final set of distributables. It's up to your team to decide where to host the build, but we recommend a source control system that has good blob support.
+
+{{< important >}}
+The .gitignore rule automatically adds the following folders to ignore for git. If using another source control system, make sure the following subdirectories of your project are ignored and not checked in:
+    
+    ```
+    \user
+    \build
+    \Cache
+    ```
+
+These files are generated locally during builds, and checking them in may cause issues for developers.
+{{< /important >}}
+
+At this point, you're ready to have your team perform local setup for either engine development or project development.
+
+### Engine development setup
+
+Engine developers will work on O3DE source code and create new distributable builds. To enable their workflows, perform the following steps on their workstations:
+
+1. Get your modified engine source code from its hosted location.
+1. Install the Python runtime and modules required by the engine. Run the following command from the directory containing engine source:
+
+    ```cmd
+    python\get_python.bat
+    ```
+1. Register the engine locally. Run the following command from the directory containing engine source:
+
+    ```cmd
+    scripts\o3de.bat register --this-engine
+    ```
+1. Get the project from its hosted location.
+1. Register the project with the _local development engine_. Run the following command from the directory containing the engine source:
+   
+   ```cmd
+   scripts\o3de.bat register -pp W:\MyProject
+   ```
+1. Follow the [project build instructions](https://o3de.org/docs/user-guide/build/) to generate any project-specific builds, such as the launcher.
+
+### Project developer setup
+
+Project developers will work on project-specific source code or use O3DE creative tools to add project content. To enable their workflows, project developers are required to _only_ pull a project from its hosted location.
```

Comment:
I suggest rewording this sentence. I'm not 100% sure what is really meant here. The emphasis of the word _only_, together with its location in the sentence, is awkward. It seems to be trying to convey that project developers _must_ pull a project from its hosted location, and no other location. If this is truly the case, then I'd start by replacing "project developers are required to only" with "project developers must only". Then, at the beginning of that sentence, in place of "To enable their workflows", I'd supply a more precise reason why they must do this.

Reply:
I hesitate calling this "Project developer setup".

This seems like information that is in addition to the engine of the "Engine development setup" section.
For users that don't need to re-build the project, they can use the prebuild binaries in the `<project-root>/install/bin/Windows/profile/Default` subdirectory.

Reply:
Leaving this for @sptramer to address in a follow up.